README.md

```diff
@@ -23,361 +23,7 @@ News](https://news.ycombinator.com/item?id=30060784#30062329) and
 [Lobsters](https://lobste.rs/s/oavgcx/prql_simpler_more_powerful_sql) earlier
 this year when it was just a proposal.
 
-## Overview
```

Comment:
If we leave this in, I'm happy to do this separately and ensure there's a "Links" section at the top.

(possibly we also want to keep a small example here, I personally prefer being able to see an example when I go to repos, and I know other similar projects got flak for being too abstract in their Readmes...)

Reply:
Yeah sure, but I think is should be shorter.